src/System.IO.FileSystem/src/System.IO.FileSystem.builds

```diff
@@ -10,22 +10,13 @@
     </Project>
     <Project Include="System.IO.FileSystem.csproj">
```

Comment:
@ericstj @JeremyKuhne @ianhays Do we want to keep live builds for the older System.IO.FileSystem library?